redis/Dockerfile

```
@@ -23,12 +23,25 @@ RUN apt-get update && \
     apt-get dist-upgrade -y && \
     rm -rf /var/lib/apt/lists/*
 
+# Make it possible to override the UID/GID/username of the user running
+ARG UID=999
+ARG GID=999
+
+# Since username and group name `redis` already exists in base image
+ENV UNAME=redis
```

Given that `uname` is something different, and `$USER` is already used to hold the username, I think we should do this:

```suggestion
ENV USER=redis
```

---

Or do we even need this ENV var to hold the username, since it will always be "redis"?